manifests/rhoai/shared/base/deployment.yaml

```diff
@@ -20,7 +20,3 @@
 - op: replace
   path: /spec/template/spec/serviceAccount
   value: rhods-dashboard
-- op: replace
-  path: /spec/template/spec/imagePullSecrets
-  value:
-    - name: addon-managed-odh-pullsecret
```

Comment:
Can you remove all references to this -- the secret does not exist on any cluster and I noticed it got copied into the NIM space from the Anaconda space... these cannot have anymore success than this one.

Reply:
Ah, I actually saw this field in NIM and I found it was added recently so I decided not to touch it. Now I updated the PR and it's removed, thanks!